src/Raven.Server/Documents/TimeSeries/TimeSeriesValuesSegment.cs

```diff
@@ -118,7 +123,7 @@ public void Initialize(int numberOfValues)
             Header->NumberOfValues = (byte)numberOfValues;
             Header->SizeOfTags = 1;
             Header->PreviousTagIndex = byte.MaxValue;// invalid tag value
-            Header->Version = SegmentVersion.Current;
+            Header->Version = SegmentVersion.V50001; // we might upgrade it later in case of duplicate timestamps
```

Comment:
Current gives you a good indication where you actually need to change things, in 2 years you will not remember that. If you have problems with naming that 'Current' than maybe call it 'Baseline' ?

Reply:
done